libc-test/build.rs

```diff
@@ -1927,6 +1928,157 @@ fn test_emscripten(target: &str) {
     cfg.generate("../src/lib.rs", "main.rs");
 }
 
+fn test_vxworks(target: &str) {
+    assert!(target.contains("vxworks"));
+
+    let mut cfg = ctest::TestGenerator::new();
+    headers! { cfg:
+               "vxWorks.h",
+               "yvals.h",
+               "nfs/nfsCommon.h",
+               "rtpLibCommon.h",
+               "randomNumGen.h",
+               "taskLib.h",
+               "sysLib.h",
+               "ioLib.h",
+               "inetLib.h",
+               "socket.h",
+               "errnoLib.h",
+               "ctype.h",
+               "dirent.h",
+               "dlfcn.h",
+               "elf.h",
+               "fcntl.h",
+               "grp.h",
+               "sys/poll.h",
+               "ifaddrs.h",
+               "langinfo.h",
+               "limits.h",
+               "link.h",
+               "locale.h",
+               "sys/stat.h",
+               "netdb.h",
+               "pthread.h",
+               "pwd.h",
+               "sched.h",
+               "semaphore.h",
+               "signal.h",
+               "stddef.h",
+               "stdint.h",
+               "stdio.h",
+               "stdlib.h",
+               "string.h",
+               "sys/file.h",
+               "sys/ioctl.h",
+               "sys/socket.h",
+               "sys/time.h",
+               "sys/times.h",
+               "sys/types.h",
+               "sys/uio.h",
+               "sys/un.h",
+               "sys/utsname.h",
+               "sys/wait.h",
+               "netinet/tcp.h",
+               "syslog.h",
+               "termios.h",
+               "time.h",
+               "ucontext.h",
+               "unistd.h",
+               "utime.h",
+               "wchar.h",
+               "errno.h",
```

Comment:
Please attach proof that this list of headers is complete.

Reply:
Pass building:
$ cargo test --no-run --manifest-path libc-test/Cargo.toml --ta\
rget x86_64-wrs-vxworks -j1
   Compiling libc v0.2.64 (/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc)
   Compiling libc-test v0.1.0 (/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/libc-test)
    Finished dev [unoptimized + debuginfo] target(s) in 13.94s

Reply:
And run pass:
[vxWorks *]# ala-diab-pb19l:/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/main-b4e94fcc89001aa7.v\
xe
ala-diab-pb19l:/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/main-b4e94fcc89001aa7.vxe
Launching process 'ala-diab-pb19l:/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/main-b4e94fcc8900\
1aa7.vxe' ...
Process 'ala-diab-pb19l:/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/main-b4e94fcc89001aa7.vxe' \
(process Id = 0x400980ac0) launched.
RUNNING ALL TESTS
PASSED 2488 tests
[vxWorks *]# ala-diab-pb19l:/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/cmsg-7dfc47db37d3828c.v\
xe
ala-diab-pb19l:/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/cmsg-7dfc47db37d3828c.vxe
Launching process 'ala-diab-pb19l:/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/cmsg-7dfc47db37d3\
828c.vxe' ...
Process 'ala-diab-pb19l:/folk/prj-rust-dev/bpang/libc_test/dev-libc/libc/target/x86_64-wrs-vxworks/debug/cmsg-7dfc47db37d3828c.vxe' \
(process Id = 0x400c9d8a0) launched.

running 5 tests
test t::test_cmsg_data ... ok
test t::test_cmsg_firsthdr ... ok
test t::test_cmsg_len ... ok
test t::test_cmsg_nxthdr ... ok
test t::test_cmsg_space ... ok

test result: ok. 5 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out